### ISSUE TITLE
bootstrap fails on machine deployment with ubuntu 16.04

### ISSUE THREAD

**Issue by konstan** [slipstream/SlipStreamClient#391]
Tested on exoscale and ec2.

Doesn't work only on ubuntu 16 with machine.

Works on orchestrator with ubuntu 16, 15, 14. 
Works on centos for orch and machine.

**Comment by konstan** [comment]
The issue is with `slipstream-node` service started by the bootstrap 

```
Done bootstrapping!

DEBUG: System supports systemd.
Created symlink from /etc/systemd/system/multi-user.target.wants/slipstream-node.service to /etc/systemd/system/slipstream-node.service.
Calling target script: systemctl start slipstream-node.service
````
It fails to start due to absence of `pkg_resources` package from setuptools.
```
root@VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3:~# systemctl status slipstream-node.service
● slipstream-node.service - SlipStream Node Executor
   Loaded: loaded (/etc/systemd/system/slipstream-node.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Wed 2018-02-21 15:11:10 UTC; 1min 14s ago
  Process: 2626 ExecStop=/opt/slipstream/client/sbin/slipstream-node stop (code=exited, status=1/FAILURE)
  Process: 2621 ExecStart=/opt/slipstream/client/sbin/slipstream-node ${DAEMON_ARGS} start (code=exited, status=1/FAILURE)
 Main PID: 2621 (code=exited, status=1/FAILURE)

Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 systemd[1]: slipstream-node.service: Main process exited, code=exited, status=1/FAILURE
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 slipstream-node[2626]: Traceback (most recent call last):
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 slipstream-node[2626]:   File "/opt/slipstream/client/sbin/slipstream-node", line 23, in <module>
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 slipstream-node[2626]:     from slipstream.command.CommandBase import CommandBase
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 slipstream-node[2626]:   File "/opt/slipstream/client/lib/slipstream/__init__.py", line 1, in <module>
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 slipstream-node[2626]:     __import__('pkg_resources').declare_namespace(__name__)
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 slipstream-node[2626]: ImportError: No module named pkg_resources
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 systemd[1]: slipstream-node.service: Control process exited, code=exited status=1
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 systemd[1]: slipstream-node.service: Unit entered failed state.
Feb 21 15:11:10 VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3 systemd[1]: slipstream-node.service: Failed with result 'exit-code'.
root@VM-8afe53b1-f081-40bd-b4a0-e1921d9317d3:~#
```

Solution. On Py3 VM after installation of Py2 make sure that setuptools is installed.

**Comment by konstan** [comment]
Was fixed directly on master with 3378ab228
